neurons/validators/src/services/task/pipeline_factory.py

```diff
@@ -224,4 +224,5 @@ def build_pipeline(self, checks: list[Check]) -> Pipeline:
         Returns:
             Configured Pipeline ready to run
         """
+        checks = self.build_checks()
```

Comment:
Why make the pipeline coupled with only the checks we have built-in? Can we let them nicely be injected instead? This probably broke all unit tests.